src/Raven.Server/Storage/Layout/LayoutUpdater.cs

```diff
@@ -73,6 +73,9 @@ private static bool TryMoveJournals(VoronPathSetting basePath, VoronPathSetting
                 var source = basePath.Combine(journalFileInfo.Name);
                 var destination = journalsPath.Combine(journalFileInfo.Name);
 
+                if (File.Exists(destination.FullPath))
+                    File.Delete(destination.FullPath);
```

Comment:
Isn't this dangerous, in the sense that we'll lose data ?

Reply:
We are about to move such file (the same name) from root dir to `Journals`. Technically it shouldn't even exists but if from some reason it's there we'll fail on `File.Move` (https://groups.google.com/forum/#!topic/ravendb/QAqneIcatsQ)

Reply:
I saw that, yes. This can happen if we crashed midway through the previous upgrade, I guess, or something similar. 
Is there any chance that there is a data loss event here by deleting this file? I can't think about any such scenario, but I want to double check this.

Reply:
I cannot as well. We might be a bit more cautious by renaming the file here instead of removing it and delete it once `StorageEnvironment` initializes successfully

Reply:
ok